### DISCUSSION TITLE
Gostaria de saber mais sobre o cenário de frontend em São Paulo e no Brasil, temos pesquisas recentes?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1773] [Discussão]
Boa tarde pessoal! Tudo bom?

Estou vendo a survey de 2018 do Forntend Brasil:
https://survey.frontendbr.com.br/

Temos esta pesquisa mais recente? Ou temos alguma outra pesquisa com a mesma pegada de 2019 ou 2020?

Muito obrigado, abraço!

**Comment by felipefialho** [comment]
Não trabalhei nisso nos últimos dois anos, é bem trabalhoso, hehe.

Mas o projeto é [open source](https://github.com/survey) 😉
